bin/cron

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env ruby
+
+$stdout.sync = true
+
+$LOAD_PATH << 'lib'
+
+require 'bundler/setup'
+require 'travis/api/app'
+require 'travis/api/v3/services/crons/start'
+
+Travis::Api::App.new
+app = Travis::API::V3::Services::Crons::Start.new(nil, nil, nil)
```

Comment:
This might be on this refactor later list, but could this code be put into a rake task? Does it make sense to still have it in a V3 service?